src/Nox.Reference.MacAddresses/AppMacAddressExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Nox.Reference.Common;
+using Nox.Reference.MacAddresses.Models;
+
+namespace Nox.Reference.MacAddresses;
+
+public static class AppMacAddressExtensions
+{
+    private const string ResourceName = "Nox.Reference.MacAddresses.json";
+
+    public static IServiceCollection AddMacAddresses(this IServiceCollection services)
```

Comment:
We will need to rename this extension method to match all other (Nox.Lib) extension methods. Will make consumer code more clean.

I'd name this one : `AddNoxMacAddresses(this IServiceCollection services)`